html/patterns/01-atoms-02-list-body-list-body/01-atoms-02-list-body-list-body.html

```diff
@@ -5,8 +5,8 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width" />
     
-    <link rel="stylesheet" href="../../css/style.css?1497365795" media="all" />
-    <link rel="stylesheet" href="../../css/pattern-scaffolding.css?1497365795" media="all" />
+    <link rel="stylesheet" href="../../css/style.css?1508604484" media="all" />
+    <link rel="stylesheet" href="../../css/pattern-scaffolding.css?1508604484" media="all" />
```

Comment:
So, the HTML files seem to still be expecting this CSS file but it was removed above. There also seems to be a bunch of other files removed. I'm curious if this was intentional